src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -419,6 +466,42 @@ def builtin_cast(
     )
 
 
+def builtin_make_const_list(
+    transformer: "PythonTaskletCodegen", node: itir.Expr, node_args: list[itir.Expr]
+) -> list[ValueExpr]:
+    di = dace_debuginfo(node, transformer.context.body.debuginfo)
+    args = [transformer.visit(arg)[0] for arg in node_args]
+    assert all([isinstance(x, (SymbolExpr, ValueExpr)) for x in args])
```

Comment:
I recently learned that you also could write
```python
all(isinstance(x, (SymbolExpr, ValueExpr)) for x in args)
```
which is actually faster, since the list has not to be created.

Reply:
Nice!